scylla-cql/src/frame/response/result.rs

```diff
@@ -66,68 +107,85 @@ pub enum ColumnType<'frame> {
     Text,
     Timestamp,
     Inet,
-    List(Box<ColumnType<'frame>>),
-    Map(Box<ColumnType<'frame>>, Box<ColumnType<'frame>>),
-    Set(Box<ColumnType<'frame>>),
-    UserDefinedType {
-        type_name: Cow<'frame, str>,
-        keyspace: Cow<'frame, str>,
-        field_types: Vec<(Cow<'frame, str>, ColumnType<'frame>)>,
-    },
     SmallInt,
     TinyInt,
     Time,
     Timeuuid,
-    Tuple(Vec<ColumnType<'frame>>),
     Uuid,
     Varint,
 }
 
+/// Collection variants of [ColumnType]. A collection is a composite type that:
+/// has no predefined size, so it is possible to add and remove values to/from it.
+///
+/// Tuple and vector are not collections because they don't have dynamic size.
```

Comment:
:pick: I'd remove negations from both sentences:

```suggestion
/// Collection variants of [ColumnType]. A collection is a composite type that:
/// has dynamic size, so it is possible to add and remove values to/from it.
///
/// Tuple and vector are not collections because they have predefined size.
```